src/Instance/InstanceDetails/InstanceDetails.js

```diff
@@ -187,6 +190,13 @@ const InstanceDetails = forwardRef(({
               classificationTypes={referenceData.classificationTypes}
             />
 
+            <Accordion
```

Comment:
@AndreiShumski2 what should happen here if the given folio build doesn't provide acquisition interfaces? Should this protected by `HasInterface` in a similar fashion to what you did here:

https://github.com/folio-org/ui-inventory/pull/1521/files#diff-eb90fc028a7da3c9660861ec34cc6ff0d67d54c079f23baa812fbdc35595e6c4R59

Reply:
Good point. Thanks. Added interface check for acq accordion